aspnetcore/host-and-deploy/blazor/webassembly.md

```diff
@@ -394,3 +394,41 @@ A Blazor WebAssembly app can be initialized with the `loadBootResource` function
 External sources must return the required CORS headers for browsers to allow the cross-origin resource loading. CDNs usually provide the required headers by default.
 
 You only need to specify types for custom behaviors. Types not specified to `loadBootResource` are loaded by the framework per their default loading behaviors.
+
+## Rename DLL files to bypass proxies, firewalls, and antivirus software
+
+Some environments block the download of *.dll* files that are normally part of a Blazor app's published deployment. To address this issue, you can rename *.dll* files to use a different file extension and update the *blazor.boot.json* file.
+
+After publishing the app:
+
+1. Open PowerShell (Windows) or a command shell (Linux or macOS) to the *wwwroot* directory of the app's published output (for example, *{CONTENT ROOT}/bin/Release/netstandard2.1/publish/wwwroot*).
```

Comment:
I think the intention is that you wouldn't normally do this manually. Instead you would automate running these scripts as part of your build.